scripts/do-release.sh

```diff
@@ -124,7 +124,7 @@ function create_pr() {
     # Apply labels separately, since each label trigger the CI separately anyway and that causes multiple runs clogging the CI up.
     dryrun gh pr edit --add-label e2e-all-k8s "${pr_url}" || echo "INFO: Didn't label 'e2e-all-k8s', continuing without it."
     dryrun gh pr edit --add-label ready-to-test "${pr_url}"
-    dryrun gh pr merge --auto --repo "${ORG}/${project}" --squash "${pr_url}" || echo "WARN: Failed to enable auto merge on ${pr_url}"
+    dryrun gh pr merge --auto --repo "${ORG}/${project}" --rebase "${pr_url}" || echo "WARN: Failed to enable auto merge on ${pr_url}"
```

Comment:
This has a risk of having rouge "apply suggestions" commits sneak in though :/

Also, do you have an example where the merge commit message sneaked in?

Reply:
See https://github.com/submariner-io/submariner/commit/896697e4c3e401371d25e90a3c2b6a4c4f40d873: the extra signed-off and co-authored by lines come from the merge commit.

Reply:
Do we actually get suggestions on bot-authored PRs?